.github/workflows/test_all_plugins.yml

```
@@ -38,8 +45,8 @@ jobs:
 
       - uses: conda-incubator/setup-miniconda@v3
         with:
-          python-version: 3.9
-          miniforge-variant: Mambaforge
+          python-version: '3.10'
+          miniforge-variant: Miniforge3
```

```suggestion
          miniforge-variant: Miniforge3
          conda-remove-defaults: "true"
```
To silence the warning
https://github.com/napari/npe2/actions/runs/12840078624